tests/test_lib_events.py

```diff
@@ -909,6 +911,169 @@ def test_02_forward_admin_request(self):
+        options = {"g": g,
+                   "request": req,
+                   "response": resp,
+                   "handler_def": {"options":
+                                       {"JSON pointer": "/detail/something",
+                                        "value": "special",
+                                        "type": "string"}
+                                   }
+                   }
+        r_handler = ResponseManglerHandler()
+        res = r_handler.do("add", options=options)
+        self.assertTrue(res)
+        self.assertEqual(json.loads(resp.data.decode("utf-8")).get("detail").get("something"), "special")
+
+        # change JSON pointer with two components
+        resp.data = """{"result": {"value": true}, "detail": {"message": "Du", "error": 1}}"""
+        options = {"g": g,
+                   "request": req,
+                   "response": resp,
+                   "handler_def": {"options":
+                                       {"JSON pointer": "/detail/message",
+                                        "value": "special",
+                                        "type": "string"}
+                                   }
+                   }
+        r_handler = ResponseManglerHandler()
+        res = r_handler.do("add", options=options)
+        self.assertTrue(res)
+        self.assertEqual(json.loads(resp.data.decode("utf-8")).get("detail").get("message"), "special")
+
+        # add the components, that do not yet exist
+        resp.data = """{"result": {"value": true}, "detail": {"message": "Du", "error": 1}}"""
+        options = {"g": g,
+                   "request": req,
+                   "response": resp,
+                   "handler_def": {"options":
+                                       {"JSON pointer": "/comp1/comp2/comp3",
+                                        "value": "1",
+                                        "type": "bool"}
+                                   }
+                   }
+        r_handler = ResponseManglerHandler()
+        res = r_handler.do("add", options=options)
+        self.assertTrue(res)
+        self.assertEqual(json.loads(resp.data.decode("utf-8")).get("comp1").get("comp2").get("comp3"), True)
+
+        # JSON pointer with more than 3 components not supported
+        resp.data = """{"result": {"value": true}, "detail": {"message": "Du", "error": 1}}"""
+        options = {"g": g,
+                   "request": req,
+                   "response": resp,
+                   "handler_def": {"options":
+                                       {"JSON pointer": "/comp1/comp2/comp3/comp4",
+                                        "value": "1",
+                                        "type": "integer"}
+                                   }
+                   }
+        r_handler = ResponseManglerHandler()
+        res = r_handler.do("add", options=options)
+        self.assertTrue(res)
+        self.assertNotIn("comp1", json.loads(resp.data.decode("utf-8")))
+
+
```

Comment:
We should also check adding a JSON pointer with 1 component.
And what happens if the "type" and "value" mismatch?

Reply:
Good idea, please add the test.

Reply:
Done in c7225471